webpack/assets/javascripts/react_app/components/hosts/powerStatus/PowerStatus.scss

```diff
@@ -0,0 +1,11 @@
+.host-power-status {
+  &.on {
+    color: #3f9c35;
```

Comment:
TBH: based on the size of this PR, please do the restructuring in another PR that we most likely can merge quickly, then you can rebase this pr again.